controllers/provider-openstack/charts/internal/cloud-provider-config/templates/cloud-provider-config.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: cloud-provider-config
+  name: {{ .Values.cloudProviderConfigName }}
```

What's the need to have this as value? 
```suggestion
  name: cloud-provider-config-cloud-controller-manager
```